### PR TITLE
Adding copyright to specs

### PR DESCRIPTION
**Pull request by usbsnowcrash**
This adds the copyright notice to our specs and also runs rubocop across them to correct whitespace and tabbing type issues.

### CODE REVIEW

File: spec/cb/client_spec.rb [hunk 1]
```diff
@@ -51,9 +60,9 @@ module Cb
         end
 
         it 'should call the api using post' do
-          expect(mock_api).to receive(:execute_http_request).
-              with(:post, base_uri, 'parts unknown', {:query=>nil, :headers=>nil, :body=>nil}).
-              and_return(Hash.new)
+          expect(mock_api).to receive(:execute_http_request)
+            .with(:post, base_uri, 'parts unknown', query: nil, headers: nil, body: nil)
+            .and_return({})
```
Comment (NicholasBusby) [inline spec/cb/client_spec.rb]: are all these other changes just running rubocop on these file as you modified them?

File: spec/cb/clients/saved_search_spec.rb [hunk 2]
```diff
@@ -103,25 +115,24 @@ module Cb
...
-              "IsDailyEmail"=>"maybe",
-              "EmailDeliveryDay"=>"YESTERDAY",
-              "JobSearchUrl"=>"blerg.com",
-              "SavedSearchParameters"=>{
-                  "Any"=>"No"}
+        stub_request(:get, uri_stem(Cb.configuration.uri_saved_search_retrieve.gsub(':did', 'xid')))
+          .to_return(body: {
+            'TotalResults' => 1,
+            'ReturnedResults' => 1,
+            'Results' =>             [{ 'DID' => 'OMG DIDs',
```
Comment (jusroberts) [inline spec/cb/clients/saved_search_spec.rb]: Lots of added indention is on purpose?

File: spec/cb/client_spec.rb (hunk 1 above)
Comment (usbsnowcrash) [inline spec/cb/client_spec.rb]: yes

File: spec/cb/clients/saved_search_spec.rb (hunk 2 above)
Comment (ausmith) [inline spec/cb/clients/saved_search_spec.rb]: SURPRISE INDENTATION! Agreed, this does seem unexpected.

File: spec/cb/models/api_response_model_spec.rb [hunk 3]
```diff
@@ -54,11 +60,12 @@ def assert_not_implemented_method_name(method_name, &expected_to_raise_error)
         end
 
         it '#set_model_properties' do
-          class FailDummyModel; def required_fields; Array.new end end
+          class FailDummyModel; def required_fields
+                                  []
+                                end end
```
Comment (ausmith) [inline spec/cb/models/api_response_model_spec.rb]: I don't think that rubocop liked this very much...

File: spec/cb/models/implementations/employee_type_spec.rb [hunk 4]
```diff
@@ -8,9 +17,9 @@ def new_model
         end
 
         it 'initializes with no args to empty string defaults' do
-          expect(new_model.code).    to eq String.new
-          expect(new_model.name).    to eq String.new
-          expect(new_model.language).to eq String.new
+          expect(new_model.code).    to eq ''
+          expect(new_model.name).    to eq ''
+          expect(new_model.language).to eq ''
```
Comment (ausmith) [inline spec/cb/models/implementations/employee_type_spec.rb]: haha, thanks rubocop!

File: spec/cb/requests/email_subscription/modify_spec.rb [hunk 5]
```diff
@@ -23,7 +32,7 @@ module Cb
         it 'should have a basic body' do
           expect(@request.body).to eq <<eos
 <Request>
-<DeveloperKey>#{Cb.configuration.dev_key.to_s}</DeveloperKey>
+<DeveloperKey>#{Cb.configuration.dev_key}</DeveloperKey>
```
Comment (ausmith) [inline spec/cb/requests/email_subscription/modify_spec.rb]: Space padding in string interpolation please :smile:

File: spec/cb/responses/application/application_form_spec.rb [hunk 6]
```diff
@@ -14,9 +24,9 @@
     it 'raises an exception' do
       response_stub.delete('Results')
 
-      expect{Cb::Responses::ApplicationForm.new(response_stub)}.
-        to raise_error(Cb::ExpectedResponseFieldMissing) do |ex|
-          expect(ex.message).to include 'Results'
+      expect { Cb::Responses::ApplicationForm.new(response_stub) }
+        .to raise_error(Cb::ExpectedResponseFieldMissing) do |ex|
+        expect(ex.message).to include 'Results'
```
Comment (ausmith) [inline spec/cb/responses/application/application_form_spec.rb]: This block looks weird before and after the rubocop change. I think the `.to ...` part can maybe go up a line?

File: spec/cb/responses/company/find_spec.rb [hunk 7]
```diff
@@ -5,62 +15,59 @@ module Cb
...
+            'HostSites' => nil,
+            'SDrive' => nil,
+            'CompanyPhotos' => { 'PhotoList' => nil },
+            'CompanyAddress' => { 'AddressList' => nil },
+            'CompanyBulletinBoard' => { 'bulletinboards' => nil },
+            'Testimonials' => { 'Testimonials' => nil },
+            'MyContent' => { 'MyContentTabs' => nil },
+            'MyPhotos' => nil,
+            'InfoTabs' =>              { 'InfoTabs' =>                 { 'InfoTab' =>                    [{ 'Name' => 'Products',
+                                                                                                            'Content' => ''
```
Comment (ausmith) [inline spec/cb/responses/company/find_spec.rb]: WOAH! What's up with this tabbing??

Comment (tkling) [inline spec/cb/responses/company/find_spec.rb]: O____O damn, extreme

File: spec/cb/responses/data_lists/data_list_response_spec.rb [hunk 8]
```diff
@@ -15,7 +25,7 @@ module Responses
         end
         it do
           expect { Cb::Responses::ResumeDataList.new(response_stub) }
-          .to raise_error(Cb::ExpectedResponseFieldMissing) do |ex|
+            .to raise_error(Cb::ExpectedResponseFieldMissing) do |ex|
```
Comment (ausmith) [inline spec/cb/responses/data_lists/data_list_response_spec.rb]: Same here, is multi-line chaining the sensible choice?

File: spec/cb/responses/employee_types/search_spec.rb [hunk 9]
```diff
@@ -1,12 +1,22 @@
...
     let(:json_hash) do
       { 'ResponseEmployeeTypes' => {
-          'EmployeeTypes' => {
-            'EmployeeType' => [{'whoa' => 'nelly'}]
-          }
+        'EmployeeTypes' => {
+          'EmployeeType' => [{ 'whoa' => 'nelly' }]
         }
       }
+      }
```
Comment (ausmith) [inline spec/cb/responses/employee_types/search_spec.rb]: Hmmm, it kinda screwed this one up.

File: spec/cb/responses/industry/search_spec.rb [hunk 10]
```diff
@@ -1,10 +1,20 @@
...
     let(:response) do
       { 'ResponseIndustryCodes' => {
-          'IndustryCodes' => {
-              'IndustryCode' => [{'like' => 'whoa'}]
-          }
+        'IndustryCodes' => {
+          'IndustryCode' => [{ 'like' => 'whoa' }]
+        }
       }
       }
```
Comment (ausmith) [inline spec/cb/responses/industry/search_spec.rb]: I don't know how much you want to fixup styling in this PR, but this one also seems weird with the double curly closing braces.

File: spec/cb/responses/resumes/language_codes_spec.rb [hunk 11]
```diff
@@ -25,8 +35,8 @@ module Responses
           response_stub['ResponseLanguageCodes']['LanguageCodes'].delete('LanguageCode')
         end
         it do
-          expect {Cb::Responses::LanguageCodes.new(response_stub)}.
-              to raise_error(Cb::ExpectedResponseFieldMissing) do |ex|
+          expect { Cb::Responses::LanguageCodes.new(response_stub) }
+            .to raise_error(Cb::ExpectedResponseFieldMissing) do |ex|
```
Comment (ausmith) [inline spec/cb/responses/resumes/language_codes_spec.rb]: More debatable multi-line chaining.

File: spec/cb/responses/resumes/resume_list_spec.rb [hunk 12]
```diff
@@ -11,8 +21,8 @@
   context 'when the Api response is missing the results field' do
     it 'raises an exception' do
       response.delete('Resumes')
-      expect{ Cb::Responses::ResumeList.new(response) }.
-          to raise_error(Cb::ExpectedResponseFieldMissing) do |ex|
+      expect { Cb::Responses::ResumeList.new(response) }
+        .to raise_error(Cb::ExpectedResponseFieldMissing) do |ex|
```
Comment (ausmith) [inline spec/cb/responses/resumes/resume_list_spec.rb]: Last one I'm going to comment on, they stand out when scrolling through the PR from here if you want to tackle them.

File: spec/cb/models/api_response_model_spec.rb (hunk 3 above)
Comment (usbsnowcrash) [inline spec/cb/models/api_response_model_spec.rb]: I dont like this very much

File: spec/cb/requests/email_subscription/modify_spec.rb (hunk 5 above)
Comment (usbsnowcrash) [inline spec/cb/requests/email_subscription/modify_spec.rb]: I swear I made this change already

**Pull request merged**